manager/testdata/notebook/README.md

```diff
@@ -0,0 +1,347 @@
+        interface: 
+          protocol: fybrik-arrow-flight
+    - dataSetID: "fybrik-notebook-sample/new-paysim-parquet"
+      flow: write
+      requirements:
+        interface:
+          protocol: fybrik-arrow-flight
+    - dataSetID: "fybrik-notebook-sample/new-paysim-parquet"
+      requirements:
+        interface:
+          protocol: fybrik-arrow-flight
+
+EOF
+```
+
+Notice that:
+
+* The `selector` field matches the labels of our Jupyter notebook workload.
+* The `data` field includes a `dataSetID` that matches the asset identifier in the catalog.
+* The `protocol` indicates that the developer wants to consume the data using Apache Arrow Flight. For some protocols a `dataformat` can be specified as well (e.g., `s3` protocol and `parquet` format).
+
+
+Run the following command to wait until the `FybrikApplication` is ready:
+
+```bash
+while [[ $(kubectl get fybrikapplication my-notebook -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
+```
+
+## Read the dataset from the notebook
+
+In your **terminal**, run the following command to print the [endpoints](../../reference/crds/#fybrikapplicationstatusreadendpointsmapkey)(*) to use for reading the data. It fetches the code from the `FybrikApplication` resource:
+```bash
+ENDPOINT_SCHEME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/orig-paysim-csv.endpoint.fybrik-arrow-flight.scheme})
+ENDPOINT_HOSTNAME=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/orig-paysim-csv.endpoint.fybrik-arrow-flight.hostname})
+ENDPOINT_PORT=$(kubectl get fybrikapplication my-notebook -o jsonpath={.status.assetStates.fybrik-notebook-sample/orig-paysim-csv.endpoint.fybrik-arrow-flight.port})
+printf "${ENDPOINT_SCHEME}://${ENDPOINT_HOSTNAME}:${ENDPOINT_PORT}"
+```
+
+* For the simplicity of the demonstration only the original dataset endpoint is been extracted.
+
+The next steps use the endpoint to read the data in a python notebook
+
+1. Insert a new notebook cell to install pandas and pyarrow packages:
+  ```python
+  %pip install pandas pyarrow numpy
+  ```
+2. Insert a new notebook cell to read the data using the endpoint value extracted from the `FybrikApplication` in the previous step:
+  ```bash
+  import json
+import pyarrow.flight as fl
+import pandas as pd
+import pyarrow as pa
+import numpy
+
+# Create a Flight client
+client = fl.connect('<ENDPOINT>')
+
+# Prepare the request to read the original data
+request = {
+    "asset": "fybrik-notebook-sample/orig-paysim-csv",
+    # To request specific columns add to the request a "columns" key with a list of column names
+    # "columns": [...]
+}
+# Send request and fetch result
+info = client.get_flight_info(fl.FlightDescriptor.for_command(json.dumps(request)))
+reader: fl.FlightStreamReader = client.do_get(info.endpoints[0].ticket)
+df: pd.DataFrame = reader.read_pandas()
```

Comment:
I think there is no need to read the `orig-paysim-csv`. We can demonstrate the write operation then to read what was written.

Reply:
Yes, I thought about this option... was not sure what's preferred ...